LDAR_Sim/src/programs/method.py

```diff
@@ -43,8 +43,14 @@
     ERR_MSG_UNKNOWN_SENS_TYPE,
 )
 
+WEATHER_ERROR = "Error: Unrecognized weather type"
+
 
 class Method:
+    TEMP = "temp"
```

Comment:
These should get moved to a constants file

Reply:
Will be addressed later.